easybuild/easyconfigs/s/scikit-lego/scikit-lego-0.6.16-foss-2022a.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'PythonBundle'
+
+name = 'scikit-lego'
+version = '0.6.16'
+
+homepage = 'https://github.com/koaning/scikit-lego'
+description = """
+We love scikit learn but very often we find ourselves writing custom transformers, metrics and
+models. The goal of this project is to attempt to consolidate these into a package that offers
+code quality/testing. 
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+github_account = 'koaning'
+
+builddependencies = [
+    ('binutils', '2.38'),
+]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('scikit-learn', '1.1.2'),
+    ('matplotlib', '3.5.2'),
+    ('umap-learn', '0.5.3'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('autograd', '1.6.2', {
+        'checksums': ['8731e08a0c4e389d8695a40072ada4512641c113b6cace8f4cfbe8eb7e9aedeb'],
+    }),
+    ('Deprecated', '1.2.14', {
+        'checksums': ['e5323eb936458dccc2582dc6f9c322c852a775a27065ff2b0c4970b9d53d01b3'],
+    }),
+    ('patsy', '0.5.3', {
+        'checksums': ['bdc18001875e319bc91c812c1eb6a10be4bb13cb81eb763f466179dca3b67277'],
+    }),
+    ('wrapt', '1.14.1', {
```

Comment:
@WilleBell We have an easyconfig for `wrapt` 1.15.0 with `foss/2022a`, any reason you're not using that as a dependency?